pkg/features/godel_features.go

```diff
@@ -72,6 +72,12 @@ const (
 	//
 	// Allows colocation
 	EnableColocation featuregate.Feature = "EnableColocation"
+
```

Comment:
load aware is a plugin, why do we need a feature gate for it ?

Reply:
Removed the featuregate and enabled LoadAwareStore by default, ptal

Reply:
Thanks for this, we can merge this PR, and leave a TODO for you (as we discussed offline).